### PR TITLE
chore: switch to pnpm

### PR DESCRIPTION
**Pull request by markphelps**
all the cool kids use [pnpm](https://pnpm.io)

https://pnpm.io/motivation

https://refine.dev/blog/pnpm-vs-npm-and-yarn/#introduction

### CODE REVIEW

File: ui/package.json [hunk 1]
```diff
@@ -77,7 +78,6 @@
     "eslint-plugin-react-hooks": "^4.6.0",
     "jest": "^29.6.1",
     "package-changed": "^3.0.0",
-    "playwright": "^1.36.0",
```
Comment (GeorgeMac) [inline ui/package.json:79]: Im probably wrong, but I thought I needed this for `mage dagger:run generate:screenshots`.
Just mentioning incase.

**Review by GeorgeMac — APPROVED** [review]
Is this going to be `yarn` all over again? 😂

**Comment by markphelps** [comment]
> Is this going to be `yarn` all over again? 😂 

The cool kids have never been wrong @GeorgeMac

**Comment by markphelps** [comment]
closing for now as i cant get it to work correctly with playwright & dagger

**Pull request closed without merging**